bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/common/MempoolRpcTest.scala

```diff
@@ -31,7 +29,7 @@ class MempoolRpcTest extends BitcoindRpcTest {
 
         val instanceWithoutBroadcast =
           BitcoindInstance.fromConfig(configNoBroadcast,
-                                      BitcoindRpcTestUtil.newestBitcoindBinary)
+                                      BitcoindRpcTestUtil.getBinary(V18))
```

Comment:
Why v18 here?

Reply:
Because on line 19 we are creating a pair of v18 nodes, if it is left at `newestBitcoindBinary` it will be expecting a v18 bitcoind binary and get a v19 bitcoind binary